### ISSUE TITLE
Carthage/Dynamic Framework Issue

### ISSUE THREAD

**Issue by istx25** [stripe/stripe-ios#232]
Not sure if this is a Stripe issue or a Carthage issue, but when I run `carthage update`, I receive this:

```
douglas ~/Workspace/linehop $ carthage update
*** Fetching SAMTextView
*** Fetching stripe-ios
*** Downloading stripe-ios at "v5.0.0"
*** Checking out stripe-ios at "v5.0.0"
*** Checking out SAMTextView at "d94fe8d45484d33c917064535fdc0adb66d228bb"
*** xcodebuild output can be found in /var/folders/n0/b6cmfvs16676c09sswtw51kr0000gn/T/carthage-xcodebuild.7fhDjq.log
*** Building scheme "SAMTextView" in SAMTextView.xcodeproj
*** Building scheme "StripeOSX" in Stripe.xcworkspace
** BUILD FAILED **


The following build commands failed:
	CompileC /Users/douglas/Library/Developer/Xcode/DerivedData/Stripe-crdxnqrzlkhflbfeqrddsypzvlgc/Build/Intermediates/Stripe.build/Release/StripeOSX.build/Objects-normal/x86_64/STPOSXCheckoutWebViewAdapter.o Stripe/Checkout/STPOSXCheckoutWebViewAdapter.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
warning: no umbrella header found for target 'StripeOSX', module map will not be generated
/Users/douglas/Workspace/linehop/Carthage/Checkouts/stripe-ios/Stripe/Checkout/STPOSXCheckoutWebViewAdapter.m:27:36: error: assigning to 'id<WebFrameLoadDelegate>' from incompatible type 'STPOSXCheckoutWebViewAdapter *__strong' [-Werror]
/Users/douglas/Workspace/linehop/Carthage/Checkouts/stripe-ios/Stripe/Checkout/STPOSXCheckoutWebViewAdapter.m:28:33: error: assigning to 'id<WebPolicyDelegate>' from incompatible type 'STPOSXCheckoutWebViewAdapter *__strong' [-Werror]
/Users/douglas/Workspace/linehop/Carthage/Checkouts/stripe-ios/Stripe/Checkout/STPOSXCheckoutWebViewAdapter.m:29:39: error: assigning to 'id<WebResourceLoadDelegate>' from incompatible type 'STPOSXCheckoutWebViewAdapter *__strong' [-Werror]
A shell task failed with exit code 65:
** BUILD FAILED **


The following build commands failed:
	CompileC /Users/douglas/Library/Developer/Xcode/DerivedData/Stripe-crdxnqrzlkhflbfeqrddsypzvlgc/Build/Intermediates/Stripe.build/Release/StripeOSX.build/Objects-normal/x86_64/STPOSXCheckoutWebViewAdapter.o Stripe/Checkout/STPOSXCheckoutWebViewAdapter.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
```

I've deleted my derived data and tried multiple times. Still nothing.

My `Cartfile` looks like:

```
github "stripe/stripe-ios" "v5.0.0"
github "linehop/SAMTextView" "master"
```

:dancer:

**Comment by istx25** [comment]
I've tried all of what you suggested before in #210 (different problem though).

**Comment by benzguo** [comment]
Thanks for the report – I'm able to reproduce the issue. If you don't need the OSX framework, you can run `carthage update --platform ios`, which should work. We'll look into fixing the OSX target.

**Comment by istx25** [comment]
Fantastic! `carthage update --platform ios` worked for me. I won't need the OS X framework. Thanks, @benzguo! :dancer:

**Comment by benzguo** [comment]
:smile:

**Comment by jflinter** [comment]
Another note, in case anyone else runs into this - I believe this issue should be fixed either way in version 5.1.4, released last night.

**Comment by istx25** [comment]
Sweet!
